app/controllers/posts_controller.rb

```diff
@@ -6,6 +6,8 @@ def index
 
   def show
     @post = Post.find(params[:id])
+    @comments = @post.comments.select { |comment|  comment.comment_id == nil }
```

Comment:
Can you tell me more about what this is doing?

Reply:
This is finding comments with no parent comments. It displays them first, then in the partial it recursively finds those comments children.

Reply:
Ah, cool. Usually, we'd want to do this using the database rather than ruby (most operations are faster in the database). But let's leave this for now since this will get changed when you refactor to use polymorphism.